CONVENTIONS.md

```diff
@@ -290,6 +290,24 @@ Resource request review history:
 * [BZ 1920159](https://bugzilla.redhat.com/show_bug.cgi?id=1920159) --
   for tracking changes in 4.7/4.8 for single-node RAN
 
+##### Allowed use of limits
+
+There is an exception process for adding limits to payload workloads. In the following
+scenarios workloads may be given permission to set limits:
+
+* Workloads that are scale-invariant and have a fixed memory or CPU cost no matter what the scale of the cluster or workload
```

Comment:
I'm not sure what "scale-invariant" means. Is that the same as "have a fixed memory or CPU cost..."?

Reply:
It is, I could just say "scale-invariant (have fixed memory or CPU usage regardless of the size of cluster, size of node, or size of workload and usage)"

Reply:
I try to leave jargon out unless the word is an important aspect of what is being explained or the text is less readable (because you have to repeat a longer phrase over and over). In this case, I would cut "scale-invariant" in favor of the simpler terms.